standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -740,7 +752,9 @@ struct CheckConstraintsResponse {
 struct AllTableConstraintsRequest {
   1: required string dbName,
   2: required string tblName,
-  3: required string catName
+  3: required string catName,
+  4: optional string validWriteIdList,
+  5: optional i64 tableId=-1 // table id
```

Comment:
You can probably remove // table id comment.

Reply:
Done